library/modules/es7.math.fscale.js

```diff
@@ -0,0 +1,40 @@
+// https://www.npmjs.com/package/ecma-proposal-math-extensions
+var $export = require('./_export')
+  // , scale = require('./es7.math.scale')
+  , f32A = new Float32Array(1);
```

Comment:
So, I should extract an `./_fscale.js` that invokes `packIEEE754` and `unpackIEEE754` from `./_typed-buffer.js`, right? If so, that means I need to extract `./_scale.js`, too, right?